master/pkg/stream/stream.go

```diff
@@ -81,24 +97,30 @@ func (s *Subscription[T]) Configure(filter func(T) bool) {
 			s.Publisher.Subscriptions = s.Publisher.Subscriptions[:last]
 			break
 		}
-	} else {
-		// Modify an existing registraiton.
-		// (just save filter, below)
-	}
+	} // else {
```

Comment:
question: is this change intended to also implement the "stop allowing subscription mods", or is that intended to come later?

Reply:
this is another case of the linter not being happy, it was flagged it as an empty else clause, I'll just strip this out and add a comment